client/src/components/gameView.js

```diff
@@ -37,12 +81,12 @@ export default class GameView extends React.Component {
         </FormControl>
 
         <FormControl component="fieldset" onChange = {(event) => this.props.onFirstPlayerChanged(event.target.value)}>
-          <FormLabel component="legend">Select First Player</FormLabel>
+          <FormLabel component="legend">First Player</FormLabel>
           <RadioGroup>
-            <FormControlLabel value={this.props.playerIds[(order + 0) % 4]} control={<Radio />} label={this.props.playerUsernames[(order + 0) % 4]} />
```

Comment:
I think you can use an array to contain orders, then, you can just use .map() here, as well as the cards component with some `if idx == 0` kind of logic.